salt/config/__init__.py

```diff
@@ -1578,6 +1586,8 @@ def _gather_buffer_space():
     'winrepo_passphrase': '',
     'winrepo_refspecs': _DFLT_REFSPECS,
     'syndic_wait': 5,
+    'jinja_line_statement_prefix': None,
```

Comment:
This looks like a mismatch with the `str` type defined above in L845. Would it be better to have `''` be the default for consistency?